src/main/java/th/ac/kmitl/science/comsci/example/nameprinters/NamePrinterBase.java

```diff
@@ -0,0 +1,13 @@
+package th.ac.kmitl.science.comsci.example.nameprinters;
+
+import java.io.PrintWriter;
+
+public abstract class NamePrinterBase implements NamePrinter {
+
+    private final String listItemTagPattern = "<li>%s</li>";
```

Comment:
This so called `Template` instead of `Pattern` in the term of word processing.

Reply:
@khemtat I'll fix this in the next push.